src/js/arrow/links.js

```diff
@@ -0,0 +1,16 @@
+// Adding pdf span to the links with class pdf
+var pdf = '<span class="pdf-link-icon no-underline-icon" aria-hidden="true">PDF</span><span class="sr-only"> (this is a pdf file)</span>';
+
+var pdflLink = document.querySelectorAll(".pdf");
```

Comment:
We could make the selector more strict by looking for links with pdf in the href like document.querySelectorAll("a[href*='.pdf']")

Reply:
I see your point. That would stylize all pdf links. But what if content team doesn't need that extra span? This way it kind of give them more control. (Although... automation is is our goal. This is tough one)